content/post/leadership-ci.md

```diff
@@ -0,0 +1,118 @@
+
+    **Pass:** This is morally good and if not handled has long term consequences.
+
+    **Fail:** This is self serving.
+
+2. Am I including everyone?
+
+    **Pass:** My ego is not driving this conversation.
+
+    **Fail:** The people in this conversation will only tell me I'm right and not push back.
+
+3. Am I hiding something?
+
+    **Pass:** The information, though painful, is known to all.
+
+    **Fail**: Yes.
+
+4. Is there transparency here?
+
+    **Pass:** The team agrees on context and can repeat it back to me.
+
+    **Fail:** Hidden misalignment (test: what do we align on).
+
+5. Am I being curious?
+
+    **Pass:** I'm asking questions that make me uncomfortable, and I'm comfortable being wrong.
+
+    **Fail:** I want my way.
+
+6. Is my team afraid to tell me things?
+
+    **Pass:** They freely and continually come to me with answers and information that they know I will not like.
+
+    **Fail:** They go to each other or people outside the team with the information, and telling me what they think I want to hear.
+
+7. Am I only communicating with the same people over and over?
+
+    **Pass:** My sphere of influence is diverse. I feel comfortable talking with anyone on the team.
+
+    **Fail:** I continually consult the same individuals (test: do I have entourage?).
+
+8. Do I feel insecure?
+
+    **Pass:** I feel empowered and am willing to take feedback and risks regardless of the outcome as it's good for the company and the customer.
+
+    **Fail:** I retreat, I am not comfortable, I am not giving up the information because I am scared of what people will think.
+
+9. Can my team do the job I hired them to do? Is the job they are hired to do
+   the job that needs to be done?
+
+    **Pass:** The team ships outcomes efficiently.
+
+    **Fail:** The team is not empowered and often stalls (test: do I often have to intervene?).
+
+10. Are you scratching an itch?
+
+    **Pass:** This is a problem that's bigger than myself.
+
+    **Fail:** It may feel good to solve this problem but only for myself and temporarily.
+
+11. Am I being judgmental?
+
+    **Pass:** Do I trust my team and their decisions?
+
+    **Fail:** Is someone speaking up and telling me that I’m being judgmental?
+
+12. Am I taking risks?
+
+    **Pass:** I feel comfortable and confident that this decision will lead to positive and fruitful outcomes.
+
+    **Fail:** I am being a pushover, and I am compromising in the wrong ways.
+
+13. Am I being manipulative?
+
+    **Pass:** I’m being honest, real, straightforward and I’m OK with the outcome and hearing ‘no’.
+
+    **Fail:** I’m intentionally using words that aren’t representative of what I’m trying to communicate.
+
+14. Am I speaking for people or letting them speak for themselves?
+
+    **Pass:** I am doing the minority of the speaking and people are disagreeing
+with my opinions.
+
+    **Fail:** I am being quoted back to myself. I am talking the majority of the
+time.
```

Comment:
Maybe add another callout to follow along in the gist if people want to see how this list evolves - how we prioritize it... etc.